toonz/sources/tnzbase/trasterfx.cpp

```diff
@@ -834,6 +836,10 @@ void TRasterFx::compute(TTile &tile, double frame,
   if (tile.m_pos != newTilePos) {
     /*-- RenderSettingsのaffine行列に位置ずれを足しこむ --*/
     TRenderSettings newInfo(info);
+    // Image size is a 0 point.  Do nothing
+    if (!(newInfo.m_cameraBox.x0 + newInfo.m_cameraBox.x1) &&
+        !(newInfo.m_cameraBox.y0 + newInfo.m_cameraBox.y1))
+      return;
```

Comment:
Please try just putting any level and preview the scene. This condition seems to always prevent the scene to be rendered.

Reply:
Odd.  The scene I usually test stuff with worked fine, but with a new scene it doesn't. Will l look further into this unless there is a better way to fix this.

Reply:
My new scene issue appears to be something else.  I think I had some odd setting that was even impacting my official OT release.  Preview made my scene black, no matter what I did.  At any rate, after reinstalling OT using a nightly to recreate my stuff folder, I tried creating a new level and turned on preview.  I am not seeing any issue.   It's working as I expect it to, unless my expectation is wrong. 

Can you send me a scene, a video or exact steps to reproduce what you are seeing so I can see what you are referring to and try to duplicate?

Reply:
It seems that you need to make the camera dpi to be slightly different from one of the toonz raster / raster level you are putting like as follows:

![image](https://user-images.githubusercontent.com/17974955/50508979-7a694600-0ac7-11e9-9817-19ca5b916a8d.png)

Having the different dpi between the camera and the level makes tile position to be with fraction part. And it is the condition to enter this part of the source.